src/main/java/environment/Player.java

```diff
@@ -0,0 +1,45 @@
+package environment;
+
+import java.util.HashMap;
+
+public class Player {
+    private static final int MAX_ITEM_LIMIT = 5;
+    private final String name;
+    private final HashMap<String, Item> items;
+
+    public Player(String name) {
+        this.name = name;
+        this.items = new HashMap<>();
+    }
+
+    public String getName() {
+        return this.name;
+    }
+
+    public String getItemList() {
+        StringBuilder toReturn = new StringBuilder("Items in backpack");
+        int count = 1;
+        for (String itemName : this.items.keySet()) {
+            toReturn.append("\n")
+                    .append(count)
+                    .append(". ")
+                    .append(itemName);
+            count += 1;
+        }
+        return toReturn.toString();
+    }
+
+    public void pickUp(Item item) {
+        this.items.put(item.getName(), item);
+    }
+
+    public void useItem(String itemName) {
```

Comment:
To standardise the names of methods in this class, maybe we can consider changing this name to "use" or add "Item" to the rest of the methods (eg pickUpItem instead of pickUp, dropItem instead of drop etc)?